lib/components/result-view/history.js

```diff
@@ -52,14 +52,16 @@ const History = observer(({ store }: { store: OutputStore }) => {
           fontSize: atom.config.get(`Hydrogen.outputAreaFontSize`) || "inherit"
         }}
       >
-        <Output
-          output={toJS(output)}
-          displayOrder={displayOrder}
-          transforms={transforms}
-          theme="light"
-          models={{}}
-          expanded
-        />
+        <div comment="This div is used to keep the output as display:block for markdown cells.">
```

Comment:
Because the `MarkdownRenderer` component of `@nteract` outputs straight html code with no wrapper, the flow like they are columns not rows. There is a way to make a wrapping div by adding `className="someString"` to MarkdownRenderer, however wrapping is not default behavior. I cannot add a `className` though because it is being rendered through the `Output` component of `@nteract`. There is a metadata tag for `Output`, however I am unaware if I can use it to add a `className` to `MarkdownRenderer`. I also can't find anything about the metadata tag in `@nteract`'s documentation.